tasks/test_results_finisher.py

```diff
@@ -95,13 +96,22 @@ async def process_async_within_lock(
 
         if self.check_if_no_success(previous_result):
             # every processor errored, nothing to notify on
+            metrics.incr(
+                "test_results.finisher",
+                tags={"status": "failure", "reason": "no_success"},
+            )
             return {"notify_attempted": False, "notify_succeeded": False}
 
-        test_instances = latest_test_instances_for_a_given_commit(
-            db_session, commit.id_
-        )
+        with metrics.timing("test_results.finisher.fetch_latest_test_instances"):
+            test_instances = latest_test_instances_for_a_given_commit(
+                db_session, commit.id_
```

Comment:
the query in `latest_test_instances_for_a_given_commit` filters by the commit report `commit_id` which uses the commit's db record id instead of its commitid

Reply:
agreed, I also got confused the first time i saw this lol